src/renderer/components/EnhancedTable/__tests__/tableUtil.spec.js

```diff
@@ -0,0 +1,134 @@
+import * as tableUtil from '../tableUtil';
+
+describe('/renderer/components/EnhancedTable/tableUtil', () => {
+  const SELECTIVE_HEAD_CELL_VALUES = {
+    NAME: 'NAME',
+    AUTOSYNC: 'AUTOSYNC',
+    URL: 'URL',
+  };
+
+  const selectivePathToData = {
+    [SELECTIVE_HEAD_CELL_VALUES.NAME]: ['name'],
+    [SELECTIVE_HEAD_CELL_VALUES.AUTOSYNC]: ['syncAll'],
+    [SELECTIVE_HEAD_CELL_VALUES.URL]: ['cloudUrl'],
+  };
+
+  const HEAD_CELL_VALUES = {
+    NAME: 'NAME',
+    URL: 'URL',
+    USERNAME: 'USERNAME',
+    STATUS: 'STATUS',
+  };
+
+  const pathToData = {
+    [HEAD_CELL_VALUES.NAME]: ['name'],
+    [HEAD_CELL_VALUES.URL]: ['cloudUrl'],
+    [HEAD_CELL_VALUES.USERNAME]: ['username'],
+    [HEAD_CELL_VALUES.STATUS]: ['status'],
+  };
+
+  describe('getTableData()', () => {
+    [true, false].forEach((isSelectiveSyncView) => {
+      it(`returns an object depends on |isSelectiveSyncView| value. |isSelectiveSyncView = ${isSelectiveSyncView}|`, () => {
```

Comment:
I like that you've noticed my use of using the pipe (|) for the parameterized parts of a test in the descriptions, but try to keep them only around what changes. In this case,

```suggestion
      it(`uses |${isSelectiveSyncView ? SELECTIVE_HEAD_CELL_VALUES : HEAD_CELL_VALUES}| as the head cell value when isSelectiveSyncView is |${isSelectiveSyncView}|`, () => {
```

Or something similar. Just make it specific to what's expected instead of too generic.

Reply:
Got it, noted. Thanks.
Fixed